app/models/concerns/password_resets_limitable.rb

```diff
@@ -17,12 +24,19 @@ def update_password_resets_requested
         self.resets_requested += 1
       end
     end
+
+    protected
+
+    def clear_reset_password_token
```

Comment:
I think we could probably use a code comment here, explaining that this is a Devise method and why we need to change it.